src/app/shared/operators/state.ts

```diff
@@ -49,3 +49,12 @@ export function mapStateData<T, U>(dataMapper: (data: T) => U): OperatorFunction
     }),
   );
 }
+
+/**
+ * Rx operator which convert observable error to "error state". To be used when the state is built "manually".
+ */
+export function mapErrorToState<T>(): OperatorFunction<FetchState<T>,FetchState<T>> {
```

Comment:
I didn't see any use of this operator, what has it been created for ?

Reply:
It has been used and then removed in a commit later. It is still used in `mapToFetchState` and may be used for other usage later so I prefer to leave it here.